doctrine/registration_form.rst

```diff
@@ -420,5 +415,11 @@ To do this, add a ``termsAccepted`` field to your form, but set its
 The :ref:`constraints <form-option-constraints>` option is also used, which allows
 us to add validation, even though there is no ``termsAccepted`` property on ``User``.
 
+Manually Authenticating after Success
+-------------------------------------
+
+If you're using Guard authentication, you can :ref:`automatically authenticate<guard-manual-auth>`
```

Comment:
We usually leave a white space before `<guard-manual-auth>`  ... but I guess this also works.

Reply:
Would keep it with space as everywhere, because of uniformity. Every other place, Ryan did it with space anyways.

Reply:
Yea, I wasn't trying to be inconsistent - I'd rather be consistent